File: Functional/Haskell/fizzbuzz.hs
module Main where

-- sort :: Integer -> String
sort x
    | x mod 15 == 0 = "fizzbuzz"
    | x mod  5 == 0 = "buzz"
    | x mod  3 == 0 = "fizz"
    | otherwise   = show x :: String

main = print $ map sqrt [1..20]

File: Functional/Elixir/fizzbuzz.ex
# Solution 1
fizzbuzz = fn
	x when rem(x, 15) == 0 -> "fizzbuzz"
	x when rem(x, 3) == 0 -> "fizz"
	x when rem(x, 5) == 0 -> "buzz"
	x -> x
end
IO.inspect Enum.map 1..20, fizzbuzz

# Solution 2
